grails-app/controllers/imod/AssessmentTechniqueController.groovy

```diff
@@ -107,6 +110,7 @@ class AssessmentTechniqueController {
 		currentLearningObjective.save()
 		}
 
+
 		// This checks when a technique is favoritized  to by a user
 		if(params.favcheck == true) {
 		// get current user object

```

Comment:
The code inside the `if` block.
between the `{}` should be tabbed one additional level.